dnsapi/dns_selfhost.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env sh
+#
+#       Author: Marvin Edeler
+#       Report Bugs here: https://github.com/Marvo2011/acme.sh/issues/1
+#	Last Edit: 17.02.2022
+
+DNS_CHALLENGE_PREFIX_ESCAPED="_acme-challenge\."
```

Comment:
don't hard code the prefix,    it's not always the value of "_acme-challenge", it can change in dns alias mode.

Reply:
@AlvinSchiller Than we should display the current prefix, because the user must know it for the right creation in the selfhost web interface.

Reply:
@Marvo2011 The log output gives already information of the fulldomain incl. prefix.